cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md

```diff
@@ -0,0 +1,36 @@
+(sec:cookbooks:allken)=
+# rift interaction in brittle-ductile coupled systems
+
+*This section was contributed by Cedric Thieulot*
+
+The setup for this experiment is based of {cite:t}`alht11,alht12,alhf13`.
+The Cartesian domain is $L_x\times L_y\times L_z=\SI{210}{\km}\times\SI{210}{\km}\times\SI{30}{\km}$
+in size (see {numref}`fig:allken_setup`).
+It is a simple 2-layer crustal model consisting of $h_{uc}=\SI{15}{\km}$ of upper crust (characterised by a Mohr-Coulomb rheology with cohesion
```

Comment:
`\SI` doesnt work anymore, we replace it with the following: (check other places below).
```suggestion
It is a simple 2-layer crustal model consisting of $h_{uc}=15 \text{ km}$ of upper crust (characterised by a Mohr-Coulomb rheology with cohesion
```

Reply:
done